application/client.js

```diff
@@ -152,7 +152,14 @@ module.exports = HttpGateway.extend({
                     try {
                         responseData = JSON.parse(responseText);
                     } catch (e) {
-                        responseData = responseText;
```

Comment:
Not without actually making the XHR ourselves instead of going through the 3rd party request library we're currently using.  If we want to start making lively more robust in its request abilities, we'll probably have to make our own custom one.